panaroo/integrate.py

```diff
@@ -191,82 +202,237 @@ def reformat_network(single_gml, output_dir, isolateName): #Generate network out
         y['longCentroidID'] = list(y['longCentroidID'])
         y['seqIDs'] = list(y['seqIDs'])
     
-    single_gml.graph.update({'isolateNames' : 'x'}) # isolateName assigned x as list(x) in merged graph
+    single_gml.graph.update({'isolateNames' : 'x'}) # isolateName from gff filename
     nx.write_gml(single_gml, output_dir + "final_graph.gml")
     
     return single_gml
 
+def merge_graphs(directories, filename, n_cpu, temp_dir, len_dif_percent, Id, family_threshold, length_outlier_support_proportion, quiet, merge_paralogs, output_dir, min_edge_support_sv, aln, alr, core):
```

Comment:
Is this copying the function from elsewhere in the code? Ideally, we don't want this to be defined twice.
The best thing to do would be to edit `merge_graphs.py` so that this is a function defined in its own right, and then `main()` just calls this function. Then you can also import and use it here without replicating code

Reply:
Defined it within merge_graphs and now importing